packages/fxa-payments-server/src/components/PaymentMethodHeader/en.ftl

```diff
@@ -3,4 +3,4 @@
 payment-method-header = Choose your payment method
 # This message is used to indicate the second step in a multi step process.
 payment-method-header-second-step = 2. { payment-method-header }
-payment-method-required = Required
+payment-method-first-approve = First you'll need to approve your subscription
```

Comment:
```suggestion
payment-method-first-approve = First you’ll need to approve your subscription
```
Please use typographic quotes for text.

Reply:
@IvoJP 
It looks like t[his commit](https://github.com/mozilla/fxa/compare/24510c38c356cfc4cf225547495e071937d5f4c2..32a290a1d60a0f1a9daba310bd1e0e0b00590a81) reverted this change. Can you push a PR with this fix before I expose this string for translation?
```payment-method-first-approve = First you’ll need to approve your subscription```